src/foam/nanos/theme/Theme.js

```diff
@@ -100,6 +100,23 @@ foam.CLASS({
       name: 'appName',
       section: 'infoSection',
     },
+    {
+      class: 'String',
+      name: 'loginDAO'
```

Comment:
So this can be removed correct?

Reply:
We still need this as a reference for token sign in put in SignIn.js.

Reply:
make the default the smebusinessregistrationdao for now - this will all have to change soon.  One will create an 'ablii' account - not associated with a business, then select an Ablii 'App' such as invoicing, which will require business creation.

Reply:
Right, makes sense.